### PR TITLE
Update parts.md

### PR DESCRIPTION
**Pull request by thomoco**
Adding details on part naming structure

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Adding details on part naming structure

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing)

All builds in Builds_1 and Builds_2 stages are always mandatory
and will run independently of the checks below:

- [ ]  Allow: All Required Checks
- [ ]  Allow: Stateless tests
- [ ]  Allow: Stateful tests
- [ ]  Allow: Integration Tests
- [ ]  Allow: Performance tests
- [ ]  Allow: All Builds
- [ ]  Allow: batch 1, 2 for multi-batch jobs
- [ ]  Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ]  Exclude: Style check
- [ ]  Exclude: Fast test
- [ ]  Exclude: All with ASAN
- [ ]  Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ]  Exclude: All with aarch64
- [ ]  Exclude: All with release
- [ ]  Exclude: All with debug
---
- [ ]  Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ]  Exclude: AST fuzzers
---
- [ ]  Do not test
- [ ]  Woolen Wolfdog
- [ ]  Upload binaries for special builds
- [ ]  Disable merge-commit
- [ ]  Disable CI cache

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 0fcaa6a1fba0fa1dc5d31b018dd75414669c3f42 with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/73805/0fcaa6a1fba0fa1dc5d31b018dd75414669c3f42/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/73805/0fcaa6a1fba0fa1dc5d31b018dd75414669c3f42/style_check.html">❌ failure</a></td></tr>
<tbody>
</table>

**Review by thevar1able — APPROVED** [review]

File: docs/en/operations/system-tables/parts.md [hunk 1]
```diff
@@ -16,7 +16,18 @@ Columns:
     - `YYYYMM` for automatic partitioning by month.
     - `any_string` when partitioning manually.
 
-- `name` ([String](../../sql-reference/data-types/string.md)) – Name of the data part.
+- `name` ([String](../../sql-reference/data-types/string.md)) – Name of the data part. The part naming structure can be used to determine many aspects of the data, ingest, and merge patterns. The part naming format is the following:
+
+```
+<partitionid>_<minimum_block_number>_<maximum_block_number>_<level>_<data_version>
```
Comment (thevar1able) [inline docs/en/operations/system-tables/parts.md:22]:
```suggestion
<partition_id>_<minimum_block_number>_<maximum_block_number>_<level>_<data_version>
```

File: docs/en/operations/system-tables/parts.md [hunk 2]
```diff
@@ -16,7 +16,18 @@ Columns:
...
+```
+
+* Definitions:
+     - `partitionid` - identifies the partition key and possibily the interval
```
Comment (thevar1able) [inline docs/en/operations/system-tables/parts.md:26]:
```suggestion
     - `partition_id` - identifies the partition key and possibly the interval
```

**Comment by alexey-milovidov** [comment]
@thomoco 

> possibily

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**